java/util/src/main/java/com/google/protobuf/util/JsonFormat.java

```diff
@@ -432,6 +491,8 @@ private void write(final CharSequence data) throws IOException {
     private final TextGenerator generator;
     // We use Gson to help handle string escapes.
     private final Gson gson;
+    private final CharSequence nullOrSpace;
+    private final CharSequence nullOrNewLine;
```

Comment:
maybe `emptyOrSpace` and `emptyOrNewLine` would be a more accurate name for these?